toolkit/scripts/containerized-build/resources/setup_functions.sh

```diff
@@ -86,15 +86,19 @@ enable_local_repo() {
     IS_REPO_ENABLED=true
     tdnf install -y createrepo
     mv /etc/yum.repos.d/local_repo.disabled_repo /etc/yum.repos.d/local_repo.repo
-    pushd /repo
-    createrepo .
-    popd
-    mkdir -p $RPMS_DIR
-    pushd $RPMS_DIR
-    createrepo .
-    popd
-    echo "-------- the local repo is enabled ---------"
-    echo "--- Package dependencies will be installed from $RPMS_DIR, /repo and upstream server ---"
+    url_list=""
+    baseurls=$(cat /etc/yum.repos.d/local_repo.repo | grep baseurl | cut -d '=' -f 2)
+    for url in $baseurls
+    do
+        url="${url#*file://}" #remove 'file://' prefix
+        mkdir -p $url
+        pushd $url
```

Comment:
- Why do we have `*file://` instead of `file://` ?
- Nit: I generally feel bash magic very very hard to grasp, so maybe we could make our intent clear.
```suggestion
    prefixToRemove="file://"
    for urlWithPrefix in $baseurls
    do
        url="${urlWithPrefix#$prefixToRemove}" #remove 'file://' prefix
        mkdir -p $url
        pushd $url
```

Reply:
Great suggestion, fixed. Also added a warning message, if the repo can't be created at the given location.